docs/eventing/getting-started.md

```diff
@@ -285,53 +197,35 @@ events based on the Cloud Event context attributes.
         name: goodbye-display
    END
    ```
-
    The command creates a trigger that sends all events of source `sendoff` to
    your event consumer named `goodbye-display`.
 
 1. Verify that the triggers are working correctly by running the following
    command:
-
-   ```sh
+   ```
    kubectl --namespace event-example get triggers
    ```
-
    This returns the `hello-display` and `goodbye-display` triggers that you
    created:
-
-   ```sh
+   ```
    NAME                   READY   REASON   BROKER    SUBSCRIBER_URI                                                                 AGE
    goodbye-display        True             default   http://goodbye-display.event-example.svc.cluster.local/                        9s
    hello-display          True             default   http://hello-display.event-example.svc.cluster.local/                          16s
    ```
+    If the triggers are correctly configured, they will be ready and pointing to the correct broker (`default`) and `SUBSCRIBER_URI`.
 
-   If the triggers are correctly configured, they will be ready and pointing to
-   the correct **Broker** (the default broker) and **SUBSCRIBER_URI** has a
-   value similar to (triggerName.namespaceName.svc.cluster.local) _exact value
-   dependent on Broker implementation_. If this is not the case, see the
-   [Debugging Guide](./debugging/README.md) to help troubleshoot the issue.
-
-You have now created all of the resources needed to receive and manage events.
-You created the `Broker`, which manages the events sent to event consumers with
-the help of triggers. In the next section, you will make the event producer that
-will be used to create your events.
-
-### Creating event producers
+    The `SUBSCRIBER_URI` has a value similar to `triggerName.namespaceName.svc.cluster.local`.
+    The exact value depends on the broker implementation.
+    If this value looks incorrect, see the [Debugging Guide](./debugging/README.md) to troubleshoot the issue.
 
-In this section you will create an event producer that you can use to interact
-with the Knative Eventing subcomponents you created earlier. Most events are
-created systematically, but this guide uses `curl` to manually send individual
-events and demonstrate how these events are received by the correct event
-consumer. Because you can only access the `Broker` from within your Eventing
-cluster, you must create a `Pod` within that cluster to act as your event
-producer.
+## Creating a pod as an event producer
```

Comment:
I'm hoping we can simplify this section. It's too low level for most users and involves a lot of steps just to send an event. I don't know if today we have a reasonable alternative but I'm hoping at some point `kn` will (or is?) support sending events.

Note that I'm not suggesting to totally get rid on this section, instead I'm suggesting to provide a simpler alternative. We could use two tabs, one "with curl", and the other one "with kn".